formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -157,21 +157,24 @@ let acceptsClosedAnonObjAsArg
     (o: < x : int, y : int >) =>
   o#x + o#y;
 
-let res = acceptsOpenAnonObjAsArg {
-  method x = 0;
-  method y = 10
-};
+let res =
+  acceptsOpenAnonObjAsArg {
+    method x = 0;
+    method y = 10
+  };
 
-let res = acceptsOpenAnonObjAsArg {
```

Comment:
There is special logic to handle this, in fact and this diff probably adds something that makes that logic be ignored.